apps/omg_watcher/lib/web/router.ex

```diff
@@ -26,17 +26,16 @@ defmodule OMG.Watcher.Web.Router do
   scope "/", OMG.Watcher.Web do
     pipe_through([:api])
 
-    get("/account/:address/balance", Controller.Account, :get_balance)
+    post("/account.get_balance", Controller.Account, :get_balance)
```

Comment:
And just one more route „/account.get_transactions” which duplicates „/transaction.all”
See: 
https://github.com/omisego/elixir-omg/pull/277/files#diff-07ff2f7ff2f357afa8dcb8852afdde2dR101